homeassistant/components/tasmota/discovery.py

```diff
@@ -117,8 +119,38 @@ async def async_device_discovered(payload, mac):
             for (tasmota_entity_config, discovery_hash) in tasmota_entities:
                 await _discover_entity(tasmota_entity_config, discovery_hash, platform)
 
+    async def async_sensors_discovered(sensors, mac):
+        """Handle discovery of (additional) sensors."""
+        platform = sensor.DOMAIN
+        await _load_platform(platform)
+
+        device_registry = await hass.helpers.device_registry.async_get_registry()
+        entity_registry = await hass.helpers.entity_registry.async_get_registry()
+        device = device_registry.async_get_device(set(), {("mac", mac)})
+
+        if device is None:
+            _LOGGER.warning("Got sensors for unknown device mac: %s", mac)
+            return
+
+        orphaned_entities = {
+            entry.unique_id: ""
```

Comment:
If you don't care about the value, why not make it a set?
```suggestion
            entry.unique_id
```